server/gitrest/packages/gitrest-base/src/utils/helpers.ts

```diff
@@ -62,13 +75,23 @@ export function getRepoManagerParamsFromRequest(request: Request): IRepoManagerP
 	const storageRoutingId: IStorageRoutingId = parseStorageRoutingId(
 		request.get(Constants.StorageRoutingIdHeader),
 	);
+
+	const isEphemeralFromRequest = request.get(Constants.IsEphemeralContainer);
+	const isEphemeralContainer: boolean =
+		typeof isEphemeralFromRequest === "undefined"
+			? false
+			: typeof isEphemeralFromRequest === "boolean"
```

Comment:
Would `request.get()` return booleans, or does it always return a string? If we know it's always a string like "true" or "false", this part `typeof isEphemeralFromRequest === "boolean"` might not be needed

Reply:
Looks like it does always return string. Simplified the logic as suggested.